app/controllers/ops_controller/diagnostics.rb

```diff
@@ -965,9 +965,15 @@ def build_backup_schedule_options_for_select
   end
 
   def database_details
-    @database = MiqDbConfig.current.options
-    db_types = MiqDbConfig.get_db_types
-    @database[:display_name] = db_types[@database[:name]]
+    @database_details = Rails.configuration.database_configuration[Rails.env]
+    @database_display_name =
+      if @database_details["database"] != "vmdb_production"
+        _("External Database")
```

Comment:
I think the differences here is what questions we ask the user... (or previously asked now that the database tab is gone)
external db:  hostname, database name, username, password
internal db:  username, password
external db on an appliance:  hostname, username, password

I don't know why this is important.  Let the user specify what they want for the database configuration.